addons/test_event_full/static/src/js/tours/wevent_register_tour.js

```diff
@@ -42,9 +42,10 @@ var discoverTalkSteps = function (talkName, fromList, reminderOn, toggleReminder
                 trigger: 'span.o_wetrack_js_reminder_text',
                 run: 'click',
             }, {
-                content: "Check Favorite is On",
+                content: `Check Favorite for "${talkName}" is Now On`,
                 trigger: 'div.o_wetrack_js_reminder i.fa-bell',
                 extra_trigger: 'span.o_wetrack_js_reminder_text:contains("Favorite On")',
+                timeout: 3000, // When test is run in parallel with others, the rpc response may not be immediate
```

Comment:
@Xavier-Do , @tde-banana-odoo 

This is the essence of the fix. 
I suspect that the bug is very difficult to reproduce because we cannot artificially slow down the connection between client and server from the chrome cli used in tests.

How about merging this and see if the bug still occurs on the heavy unforgiving nightly builds?

Reply:
I just saw it's back on 15.3 too (which makes sense)... So I'll cherry pick this in 15.3.

Reply:
The question is, is this step slow too when the tests doesn't fail?

Reply:
I tried to check in previous log but we have no info because of freezetime...

Reply:
Ok, so i've enabled the ticks on freezetime and tested locally


2021-12-01 10:00:04,977 52342 ... Tour wevent_register: step 'Go on "What This Event Is All About" talk in List (trigger: a:contains("What This Event Is All About"))' succeeded 
2021-12-01 10:00:04,993 52342 ... Tour wevent_register: step 'Check Favorite is on (trigger: div.o_wetrack_js_reminder i.fa-bell)' succeeded 


2021-12-01 10:00:05,131 52342 ... Tour wevent_register: step 'Set Favorite (trigger: span.o_wetrack_js_reminder_text)' succeeded 
2021-12-01 10:00:05,142 52342  ...Tour wevent_register: step 'Check Favorite is On (trigger: div.o_wetrack_js_reminder i.fa-bell)' succeeded 

Looks like the step is quite fast most of the time it is strange that the timeout fixes it.

Reply:
Doesn't explain why it's random? If it's fast most of the time it won't fail (what we experience locally/on single threads) and while it fails when the server is fully-loaded?

Reply:
We would expect something taking around 800ms to sometime take more than one second, but here it's usually taking 10 ms if I'm right. It is stranges that it takes more than 1000 ms from time to time. 

I pushed a branch without the fix and it's not failling anymore, :/ also the test broke ~10 times yesterday but not even once tonight. Strange

Reply:
Have you seen somewhere more than 1000 ms for this step?

Maybe + 200ms is already too long if it expects to see the changes "immediately" (when it gets the chance, which can be after 10ms in these environments without events).
I'd like to make sure it's not only related to server load before looking into a possibly random long computation.

Would you prefer we set a timeout of only 250ms in this PR and run it in real conditions to make sure it is the only problem?

Reply:
No with 1000 ms I'm talking about the default RUNNING_TOUR_TIMEOUT but it is actually 10000 not 1000 so you are actually decreasing the timeout here ;)